src/error.rs

```diff
@@ -95,6 +95,8 @@ pub enum Error {
     InvalidUpscaleDesiredUploaderVmCount,
     #[error("Options were used that are not applicable to a bootstrap deployment")]
     InvalidUpscaleOptionsForBootstrapDeployment,
+    #[error("The inventory counts do not match")]
+    InventoryCountsMismatch,
```

Comment:
Not a big deal, but I would recommend making this error a bit more specific. It could be referring to almost anything.